src/vm/jitinterface.cpp

```diff
@@ -9968,28 +9968,11 @@ void* CEEInfo::getPInvokeUnmanagedTarget(CORINFO_METHOD_HANDLE method,
         MODE_PREEMPTIVE;
     } CONTRACTL_END;
 
-    void* result = NULL;
-
-    if (ppIndirection != NULL)
-        *ppIndirection = NULL;
-
-#ifndef CROSSGEN_COMPILE
-    JIT_TO_EE_TRANSITION();
-
-    MethodDesc* ftn = GetMethod(method);
-    _ASSERTE(ftn->IsNDirect());
-    NDirectMethodDesc *pMD = (NDirectMethodDesc*)ftn;
-
-    if (pMD->NDirectTargetIsImportThunk())
-    {
-    }
-    else
-    {
-        result = pMD->GetNDirectTarget();
-    }
+    // Not expected to work due to multicore and tiered JIT potentially needing
+    // to call managed cctors
+    _ASSERTE(FALSE);
 
-    EE_TO_JIT_TRANSITION();
-#endif // CROSSGEN_COMPILE
+    void* result = NULL;
```

Comment:
Nit: It would be nice to initialize the `*ppIndirection` to NULL:

```
 ...
_ASSERTE(FALSE);

 if (ppIndirection != NULL)
        *ppIndirection = NULL;
  return NULL;
```

Reply:
Done.